s30576-pj-NatanBrzozowski/zad_kolokwium_powtorzenie.txt

```diff
@@ -111,8 +108,8 @@ Jak wypisac wszystkie pliki w biezacym katalogu, ktorych nazwa zaczyna się od a
 |-----------------------------|
 |      ODPOWIEDZ/KOMENTARZ:   |
 |-----------------------------|
-ls -l al*
-ls -l | grep '^al'
+ls -l al* | grep -v /
```

Comment:
Lepiej ale ten slash to sam się tam magicznie nie pojawi. Trzeba do `ls` coś jeszcze dodać